src/ResourceManager/Profile/Commands.Profile/help/Enable-AzureRmAlias.md

```diff
@@ -86,13 +86,13 @@ Accept wildcard characters: False
 
 ### -Scope
 Indicates what scope aliases should be enabled for. 
-Default is 'Process'
+Default is 'Local'
```

Comment:
This seems like a breaking change.  It is a good breaking change, but anyone who has written a script that calls 'Enable-AzureRmALias' will have different behavior.  What would you think of leaving the defautl as is?